tests/integration/checkpoint.bats

```diff
@@ -126,7 +126,18 @@ function simple_cr() {
 	done
 }
 
-@test "checkpoint and restore " {
+@test "checkpoint and restore" {
+	simple_cr
+}
+
+@test "checkpoint and restore (bind mount, destination is symlink)" {
+	mkdir -p rootfs/real/conf
+	ln -s /real/conf rootfs/conf
```

Comment:
This is not blocking, but just curious:
why an absolute path for `/real/conf`? I see we're using the same path in `mounts.bats`, so it works, but it's not totally clear to me why we're using this path rather than somewhere in a tmpfs (or at least something test-specific)

Reply:
Hi @haircommander 
Commit [runc run: resolve tmpfs mount dest in container scope](https://github.com/opencontainers/runc/commit/637f82d6b8b45a99519b863f8cf7461760da7a97)  changes behavior of tmpfs mount and bind mount, when the destination is symlink.
 -  changes of bind mount breaks the functionality of runc checkpoint. This is why we need this path.
 -  changes of tmpfs mount does not break anything. Since tmpfs mount is fully owned by container's mnt namespace.

Reply:
thanks for the explanation, makes sense